### DISCUSSION TITLE
Is "Arduino Nano Every" supported?

### DISCUSSION THREAD

**Discussion by pischky** [khoih-prog/TimerInterrupt#27] [Ideas]
Hello,

Looks like a very interesting library.

Is the Arduino Nano Every supported ?
It uses a ATmega4809 that has very nice features like Von Neumann architecture.

Thanks, Martin

**Comment by khoih-prog** [comment]
You have to use this [**megaAVR_TimerInterrupt**](https://github.com/khoih-prog/megaAVR_TimerInterrupt) of mine for megaAVR architecture.

**Reply by pischky** [reply]
Thanks, I will try...
